ScoutSuite/providers/aws/services/iam.py

```diff
@@ -152,18 +154,16 @@ def parse_groups(self, group, params):
         self.groups[group['id']] = group
 
     ########################################
-    ##### Managed policies
+    # Managed policies
     ########################################
 
     def parse_policies(self, fetched_policy, params):
         """
         Parse a single IAM policy and fetch additional information
         """
         api_client = params['api_client']
-        policy = {}
-        policy['name'] = fetched_policy.pop('PolicyName')
-        policy['id'] = fetched_policy.pop('PolicyId')
-        policy['arn'] = fetched_policy.pop('Arn')
+        policy = {'name': fetched_policy.pop('PolicyName'), 'id': fetched_policy.pop('PolicyId'),
+                  'arn': fetched_policy.pop('Arn')}
```

Comment:
I would keep this as it was for consistency since the `parse_...()` methods in the other services are all written like this.

Reply:
I would personnaly disagree, as it would be better to do it the "right" way(PEP8 compliant) while we're working on it and eventually open a PR to change the remaining ones.

Reply:
Yeah, I agree with @zer0x64 for this kind of issue, as I've written on another comment.

Reply:
@Aboisier @misg Do you feel like we should change these for now and the rest later or change all of them at the same time later on?

Reply:
This PR is already painful enough. Let's change this iteratively as we migrate the services to the new architecture. What do you guys think?